### PR TITLE
Revert "fix list generics"

### PR DESCRIPTION
**Pull request by Flaque**
Reverts getroomservice/browser#30

### CODE REVIEW

**Comment by Flaque** [comment]
Whoops go too excited about this.

File: src/ListClient.ts [hunk 1]
```diff
@@ -4,7 +4,9 @@ import invariant from 'tiny-invariant';
 import { LocalBus } from './localbus';
 import { ListInterpreter, ListMeta, ListStore } from '@roomservice/core';
 
-export class InnerListClient<T extends any> implements ObjectClient {
```
Comment (Flaque) [inline src/ListClient.ts:7]: @kcking if you do this, it both won't match the map client, where you pass in the whole object instead of the property value.

Comment (kcking) [reply src/ListClient.ts:7]: oh i see, what if we passed `T[number]` to all of the `ListInterpreter` methods?

Comment (kcking) [reply src/ListClient.ts:7]: (and reverted back to `T extends Array<any>`)

Comment (Flaque) [reply src/ListClient.ts:7]: > oh i see, what if we passed T[number] to all of the ListInterpreter methods?

That should work :+1:

**Review by kcking — APPROVED** [review]

**Pull request merged**